.travis.yml

```diff
@@ -57,11 +58,11 @@ install:
 
 script:
     - make configure
-    - ./configure --enable-double --with-cuda-arch=sm_20 --enable-marklines --with-python --enable-paranoid
+    - ./configure --enable-double --with-cuda-arch=sm_30 --enable-marklines --with-python --enable-paranoid
```

Comment:
it seems that CudaThreadSynchronize has been deprecated and replaced by CudaDeviceSynchronize in sm_30
thus --enable-paranoid catches the depreciated warning and crashes.